zerver/actions/presence.py

```diff
@@ -64,7 +64,11 @@ def consolidate_client(client: Client) -> Client:
 
 @statsd_increment("user_presence")
 def do_update_user_presence(
-    user_profile: UserProfile, client: Client, log_time: datetime.datetime, status: int
+    user_profile: UserProfile,
+    client: Client,
+    log_time: datetime.datetime,
+    status: int,
+    force_send_update: bool = False,
```

Comment:
It seems worth making this last parameter require being passed as a kwarg, for readability in the caller.